f8a_worker/workers/githuber.py

```diff
@@ -100,8 +95,8 @@ def execute(self, arguments):
                 return result_data
 
         token = self.configuration.GITHUB_TOKEN
-        if not token:
-            if self._rate_limit_exceeded(github.Github()):
+        if token == 'not-set':
```

Comment:
I would rather like to have a method in the configuration class for this not to spread these specific values in sources.